.github/workflows/gh-registry.yml

```diff
@@ -2,6 +2,40 @@ name: Publish strongbox-examples to the GitHub Package Registry
 on:
   [ push ]
 jobs:
+  publish-hello-strongbox-ant-ivy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: mvn -f .gh-deploy/pom.xml install
+        working-directory: 'hello-strongbox-ant-ivy'
+      - id: timestamp
+        run: echo "::set-output name=current::$(date '+%Y%m%d-%H%M')"
+        # append timestamp to package version to avoid collision
```

Comment:
Included and pointed to a `pom.xml` for simplicity / saves us having to configure ant + ivy and generate pom from scratch via `ivy:makepom`.

Reply:
Interesting approach. I also like how you've hidden this away in the `.github` directory so that newbies looking at the examples don't get confused.

Thanks! :+1: